### DISCUSSION TITLE
Is it possible to split multipage images with Adjoin, not starting at 0?

### DISCUSSION THREAD

**Discussion by valdier1** [ImageMagick/ImageMagick#5095] [Help]
I'm running a command to modify all images in a directory:

magick *.tif -set filename:f "%[t]_" +adjoin "tifsplit\%[filename:f].%03d"

The problem is, the %03d counter doesn't reset per file. It continues counting up for every split. Is there a way to get it to do:

a.000
a.001
b.000
b.001
b.002
c.000

instead of:
a.000
a.001
b.002
b.003
b.004
c.005

Also is there are a way to get adjoin to start at 1 instead of 0?

**Comment by snibgo** [comment]
Within a single IM command, it won't do what you want. A solution is to loop through files in a shell loop, calling IM once for each input file.

> Also is there are a way to get adjoin to start at 1 instead of 0?

For example, use "-scene 3" to start the count at 3.

**Reply by valdier1** [reply]
ahhh, ok, that is unfortunate, but I suppose will have to write up some batch files to manage it. Thank you for the tip on scene!

**Reply by valdier1** [reply]
I wasn't unfortunately able to figure out how to get it to output the files correctly, so I think I will have to use a different tool for this one (getting it to rename and output to a different directory is beyond my batch scripting knowledge :)
